### ISSUE TITLE
Another recharts

### ISSUE THREAD

**Issue by yihui** [yihui/recharts#11]
Unfortunately someone else came up with the same name: http://recharts.org but our recharts still remains the top one in google search.

**Comment by diegocgaona** [comment]
I hope to see your package with more graph types from echarts. They have very nice graphs!!
Thanks for your work!

**Comment by yihui** [comment]
Thanks for the feedback! This was just one of my side projects, and I don't think I'll have any spare time for it until the summer.

**Comment by diegocgaona** [comment]
Off course!! Thanks for your dedication!
And thanks for the incredible knitr!!!

**Comment by YuQuanGang** [comment]
感谢提供这么好用的R包，请问一下，你和“Taiyun”是什么关系，为什么你和他的recharts包名一致，但作为用户只能选其一，同时也希望能够为用户提供更多的展示效果，谢谢！

**Comment by yihui** [comment]
Taiyun 是我的八拜之交。我这边只是自己的一个简单试验而已，他那边的是正品。

**Comment by c101088** [comment]
作者打算提供echart3.0及以上版本的接口吗？y轴的inverse属性用不了很难受呀~~~~:)

**Comment by yihui** [comment]
抱歉，短期内还没有填坑计划……

**Comment by c101088** [comment]
已经做的很好啦，大赞~~

**Comment by GitHunter0** [comment]
Hey @yihui , have you seen this [echarts project](https://github.com/JohnCoene/echarts4r)? Maybe you could work together. It would be really awesome to have `Apache ECharts` fully working in R

**Comment by yihui** [comment]
@GitHunter0 Yes, I have seen it. I'm afraid that I don't have time to contribute anything in the foreseeable future (although I'd love to). This repository of mine is really just a simple prototype from long time ago, and I don't have a plan to invest any serious effort into it. Thanks for your info!

**Comment by GitHunter0** [comment]
@yihui , I understand man, you contribute in so many awesome projects that I can imagine how unbelievably busy you are. Keep the great work!
Fortunately `echarts4r` seems to have a solid group of contributors, so let's cheer for it

**Comment by skq642877841** [comment]
作者好，网上传了很多R和echarts结合的包，但是打包的包，功能定制有点差，个人比较喜欢echarts参数用R list的方式生成交互表，但是试来试去好像只有这个包有该方式，但是最近在生成toolbox的时候不显示，请问是该包不支持还是其他原因，或者有其他包通用可以用R list（echarts的原始参数）的方式生成吗？
